client/app/components/preprint-form-header/template.hbs

```diff
@@ -3,7 +3,12 @@
 <!-- template-lint block-indentation=false -->
 
 <div class="section-header">
-    {{name}}
+    {{#if isOpen}}
+        {{displayName}}
+    {{else}}
+        <a href="#">{{displayName}}</a>
```

Comment:
Let's not add a link when there is none, this will also add # to url and may have unintended consequences in different browser like scrolling to top. You can use the pointer class to have a pointer cursor. Also a better indication of open and closed may be to use carets right or down depending on position